src/assets/a11y-viewer.css

```diff
@@ -105,6 +110,14 @@ select:focus {
 }
 
 /* checkbox */
+@media screen and (min-width: 1055px) {
+    .checkbox {
+        width: 16%;
+        float: left;
```

Comment:
All `.app-header` direct child elements already have `float: left`, so this is redundant I think?